<launch>
	<!-- args="x y z y p r frame1 frame2 hz" -->
	<!-- front strafe -->
	<node pkg="tf" type="static_transform_publisher" name="thruster0_tf" args="0.3 0 0 3.14159265 0 0 /seabee3/base_link /seabee3/thruster0 1" />
	<!-- fwd left -->
	<node pkg="tf" type="static_transform_publisher" name="thruster1_tf" args="0 0.2 0 -1.57079633 0 0 /seabee3/base_link /seabee3/thruster1 1" />
	
	<!-- depth left -->
	<node pkg="tf" type="static_transform_publisher" name="thruster2_tf" args="0 0.1 0 0 0 1.57079633 /seabee3/base_link /seabee3/thruster2 1" />
	<!-- fwd right -->
	<node pkg="tf" type="static_transform_publisher" name="thruster3_tf" args="0 -0.2 0 -1.57079633 0 0 /seabee3/base_link /seabee3/thruster3 1" />
	
	<!-- depth right -->
	<node pkg="tf" type="static_transform_publisher" name="thruster4_tf" args="0 -0.1 0 0 0 1.57079633 /seabee3/base_link /seabee3/thruster4 1" />
	<!-- back strafe -->
	<node pkg="tf" type="static_transform_publisher" name="thruster5_tf" args="-0.3 0 0 0 0 0 /seabee3/base_link /seabee3/thruster5 1" />
	
	<node pkg="seabee3_physics" type="seabee3_physics" name="seabee3_physics" output="screen" />
</launch>